### ISSUE TITLE
Add channel for Rubocop 1.24.1

### ISSUE THREAD

**Issue by dgholz** [codeclimate/codeclimate-rubocop#308]
Hi, we'd like to update the version of Rubocop we install to match the latest available version (`1.24.1`). I don't see a matching `channel` branch on this repo, nor a matching engine in https://github.com/codeclimate/codeclimate/blob/master/config/engines.yml#L197. Please add support for version `1.24.1` of Rubocop.

**Comment by BrianHawley** [comment]
You can base it on channel/beta after #309 is merged.

Also, after #309 is merged, you can use the beta channel to keep up to date with the latest rubocop.

**Comment by dgholz** [comment]
> You can use the "beta" channel to keep up to date with the latest rubocop.

Thanks @BrianHawley but we want Code Climate to run the same version of Rubocop as we have in our project's `Gemfile` on the latest commit on the master branch, so that devs working locally can run the same version & see the same Rubocop reports as they see in Code Climate. If Code Climate updates to a newer version of Rubocop before we can bump it in our `Gemfile`, then they might see a difference & spend time raising internal tickets or investigating it themselves.

I'd much prefer it if we could bump the version in our `Gemfile` and our `.codeclimate.yml` to keep them in sync, in a single commit.

**Comment by BrianHawley** [comment]
I can't create channels, so that's one. There's also a rubocop-rails bug fix release that the PR didn't include, though that only affects fixing rather than checking, which would require another PR. Another is that 1.25.0 is out today, so I'm going to make a new PR to update beta, which will also include that rubocop-rails update.

**Comment by dgholz** [comment]
@BrianHawley thanks, could you get someone who can create channels to unblock your PR? I see that Rubocop have released `1.25.0` and I'd be quite excited to let our devs know that the version of Rubocop they install locally from the Gemfile will match the version run on Code Climate. If I'd switched to the beta channel, we'd have had a mismatch until I found out about `1.25.0` & bumped it myself.
